src/lib/homestore_backend/gc_manager.cpp

```diff
@@ -33,6 +44,13 @@ GCManager::GCManager(HSHomeObject* homeobject) :
             on_gc_task_meta_blk_found(std::move(buf), voidptr_cast(mblk));
         },
         nullptr, true);
+
+    auto gcmgr_log_level = sisl::logging::GetModuleLogLevel("gcmgr");
+    if (gcmgr_log_level == spdlog::level::level_enum::err) {
+        // if we do not set gcmgr log level throught start command line(for exampel, --log_mods gcmgr:info), set it to
+        // debug level by default
+        sisl::logging::SetModuleLogLevel("gcmgr", spdlog::level::debug);
+    }
```

Comment:
if we do not set the log level of gcmgr, we set it to debug by default.
otherwise, we do nothing and use the log level set by command line.
so, we do not make any change to config map
@xiaoxichen @Besroy